### DISCUSSION TITLE
Enable Nape repository issues

### DISCUSSION THREAD

**Discussion by T1mL3arn** [HaxeFlixel/flixel#2433] [General]
Since your fork of [Nape](https://github.com/HaxeFlixel/nape-haxe4) is the only(?) active fork at this time, consider enabling issues for the repository.

**Comment by Geokureli** [comment]
Not a bad idea, but until an owner does this, feel free to talk about haxe-nape issues here.

**Comment by Gama11** [comment]
The idea was that we don't really want to maintain it in terms of bugfixes or even new features, it's really just about compile fixes to keep it working with new Haxe versions.

**Reply by T1mL3arn** [reply]
> it's really just about compile fixes

How can I compile Nape on Windows? Previously there was sort-of-instruction for *nix with some custom preprocessor. Did this become any easier?

**Reply by Gama11** [reply]
No.

**Reply by Geokureli** [reply]
It doesn't have all the same features, but you may want to give [Echo](https://austineast.dev/echo/) by AustinEast a try
